tests/app/main/views/test_service_updates.py

```diff
@@ -211,21 +180,12 @@ def test_should_redirect_to_update_page(self, data_api_client):
             }
         )
 
-        self.assertEquals(302, response.status_code)
-        self.assertIn(
-            'http://localhost/admin/service-updates',
-            response.location)
-        self.assertIn(
-            'acknowledged=false',
-            response.location)
-        self.assertIn(
-            'audit_date=2010-01-05',
-            response.location)
-
-        data_api_client.acknowledge_audit_event.assert_called(
-            audit_event_id=123,
-            user='test@example.com'
-        )
+        assert response.status_code == 302
+        assert 'http://localhost/admin/service-updates' in response.location
+        assert 'acknowledged=false' in response.location
+        assert 'audit_date=2010-01-05' in response.location
+
+        data_api_client.acknowledge_audit_event.assert_called_with("123", 'test@example.com')
```

Comment:
The assortment of single and double quotes here bugs me for no real reason.

By your own earlier argument, would it be better to look at `call_args_list` here rather than relying on `assert_called_with`?  Swapping out all of those might be more than we need for this PR though.